src/main/java/dk/kb/labsapi/SolrExport.java

```diff
@@ -227,6 +239,41 @@ private StreamingOutput streamExportJSON(
         };
     }
 
+    /**
+     * Export Solr response as raw text.
+     * This can be useful, when loading data into a text analysis tool as Voyant.
+     */
+    private StreamingOutput streamExportTXT(
+            SolrParams request, String query, Set<String> fields, long max, Set<STRUCTURE> structure,
```

Comment:
The `query` and the `format` parameters are not used for anything, so they should not be in the method signature.